src/ai/backend/manager/models/gql.py

```diff
@@ -2256,7 +2256,9 @@ async def resolve_legacy_compute_session(
             access_key=access_key,
             status=status,
         )
-        matches = await loader.load(sess_id)
+
+        # Since sess_id is declared as a string type, we have to convert this to UUID type manually.
+        matches = await loader.load(uuid.UUID(sess_id) if isinstance(sess_id, str) else sess_id)
```

Comment:
```suggestion

        # Since sess_id is declared as a string type, we have to convert this to UUID type manually.
        match sess_id:
            case str():
                sid = SessionId(uuid.UUID(sess_id))
            case uuid.UUID():
                sid = SessionId(sess_id)
            case SessionId():
                sid = sess_id
            case _:
                raise ValueError
        matches = await loader.load(sid)
```
the parameter should be `SessionId` type

Reply:
Changing sess_id to `SessionId` will require a GQL schema modification (str -> UUID).  
I modified this to prevent schema change backports.

Reply:
> This does not require to change schema at all. the loader function `LegacyComputeSession.batch_load_detail()` accepts a key parameter as a sequence of `SessionId` and it is highly recommended to match the type here.

To ensure that `sess_id` is received as `UUID` type, we need to modify the below schema as shown.

```
legacy_compute_session(sess_id: String!, domain_name: String, access_key: String): LegacyComputeSession
```

```
legacy_compute_session(sess_id: UUID!, domain_name: String, access_key: String): LegacyComputeSession
```

Reply:
My intention was to convert the type of `sess_id` to `SessionId`. This is resolved.